tests/.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source "/etc/os-release" || source "/usr/lib/os-release"
+
+tests_repo="${tests_repo:-github.com/kata-containers/tests}"
```

Comment:
Hi @chavafg - I appreciate this may be somewhat time-consuming, but I think we're going to have to update *most* [1] of the mentions of `https://github.com/kata-containers/tests` (and `github.com/kata-containers/tests` in `.go` files) to refer instead to https://github.com/kata-containers/kata-containers/tree/master/tests", etc.

Also, as we copy files from the stand-alone repos into this one, I think we should add a tag to the stand-along repo to make it easier to compare history if we need to.

---

[1] - I say "most" rather than "all" because we probably don't want to update URLs referring to specific issues and PRs as they won't exist in this repo.